cvm-attestation/How_to_read_attestation_report.md

```diff
@@ -0,0 +1,150 @@
+2. Open Command Prompt or PowerShell.
+3. Run the following command to install Git:
+  ```powershell
+  winget install --id Git.Git -e --source winget
+  ```
+
+
+## Installing the CVM Attestation Tools
+1. Open PowerShell as an administrator.
+2. Clone the cvm-attestation-tools repository:
+  ```powershell
+  git clone https://github.com/Azure/cvm-attestation-tools.git
+  ```
+3. Navigate to the cvm-attestation folder:
+  ```powershell
+  cd cvm-attestation-tools/cvm-attestation/
+  ```
+4. Install the necessary dependencies by running the installation script:
+  ```powershell
+  .\install.ps1
+  ```
+5. To retrieve the attestation report, run the following command:
+  ```powershell
+  .\read_report.ps1
+  ```
+  > **NOTE:** Ensure you run PowerShell as an administrator because the tool requires elevated privileges to access the Virtual TPM.
+
+
+# Linux
+The following instructions are for CVMs running Linux.
+
+1. Open a terminal.
+2. Clone the cvm-attestation-tools repository:
+  ```bash
+  git clone https://github.com/Azure/cvm-attestation-tools.git
+  ```
+3. Navigate to the cvm-attestation folder:
+  ```bash
+  cd cvm-attestation-tools/cvm-attestation/
+  ```
+4. Install the necessary dependencies by running the installation script:
+  ```bash
+  sudo ./install.sh
+  ```
+5. To retrieve the attestation report, run the following command:
+  ```bash
+  sudo ./read_report.sh
+  ```
+  > **NOTE:** We use `sudo` to run the script with root privileges because the tool requires elevated permissions to access the Virtual TPM.
+
+## Reading TCB Version
+This tool displays the different TCB versions in HEX format (Big Endian) as well as in a human-readable format.
+
+### Breakdown Example
+```
+Current TCB: DB16000000000004
+  Microcode:   219
+  SNP:         22
+  TEE:         0
+  Boot Loader: 4
+```
+
+### Different TCB versions from Logs
+The logs provide various TCB versions, each representing a different state of the Trusted Computing Base. Below is an example of how these versions are logged:
+- **Current TCB version**: The TCB version currently in use.
+- **Reported TCB version**: The TCB version reported by the attestation report.
+- **Committed TCB version**: The TCB version that has been committed.
+- **Launched TCB version**: The TCB version at the time of the VM launch.
+
+These versions help in understanding the security posture and the integrity of the virtual machine environment.
+```
+2025-01-24 19:16:01,485 - log_snp_report - INFO - Current TCB version: DB16000000000004
+2025-01-24 19:16:01,485 - log_snp_report - INFO - Reported TCB version: D315000000000004
+2025-01-24 19:16:01,485 - log_snp_report - INFO - Committed TCB version: D515000000000004
+2025-01-24 19:16:01,485 - log_snp_report - INFO - Launched TCB version: D515000000000004
+```
+> By comparing these versions, we can verify that the TCB of the machine running the CVM aligns with AMD's released versions.
+
```

Comment:
I would suggest adding another section after this ## TCB Version format. Maybe you can refer to AMDs public document for this.